README.md

```diff
@@ -88,9 +88,15 @@ To configure AIProxy, see the instructions [here](#aiproxy).
 1. Open your Swift project in Xcode.
 2. Go to `File` ->  `Add Package Dependency`.
 3. In the search bar, enter [this URL](https://github.com/jamesrochabrun/SwiftOpenAI).
-4. Choose the version you'd like to install.
+4. Choose the version you'd like to install (see the note below).
 5. Click `Add Package`.
 
+Note: Xcode has a quirk where it defaults an SPM package's upper limit to 2.0.0. This package is beyond that
+limit, so you should not accept the defaults that Xcode proposes. Instead, enter the lower bound of the
+[release version](https://github.com/jamesrochabrun/SwiftOpenAI/releases) that you'd like to support, and then
+tab out of the input box for Xcode to adjust the upper bound. Alternatively, you may select `branch` -> `main`
+to stay on the bleeding edge.
+
```

Comment:
@jamesrochabrun I snuck this note in, per the discussion here: https://github.com/jamesrochabrun/SwiftOpenAI/issues/60 . Please let me know if you'd like to keep it